src/atoms/a.jsx

```diff
@@ -0,0 +1,51 @@
+import ClassNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const propTypes = {
+    children: PropTypes.node,
+    className: PropTypes.string,
+    onClick: PropTypes.func,
+    onKeyDown: PropTypes.func,
+    style: PropTypes.shape({}),
+    tabIndex: PropTypes.number,
+};
+const defaultProps = {
+    children: undefined,
+    className: undefined,
+    onClick: () => {},
+    onKeyDown: () => {},
+    tabIndex: -1,
+    style: {},
+};
+
+function A(props) {
+    const {
+        children,
+        className,
+        onClick,
+        onKeyDown,
+        style,
+        tabIndex,
+    } = props;
+    const bemBlockName = 'a';
+    const containerClassName = ClassNames('ui', bemBlockName, className);
+
+    return (
+        <span
+            className={containerClassName}
+            onClick={onClick}
+            onKeyDown={onKeyDown}
+            role="button"
+            tabIndex={tabIndex}
+            style={style}
+        >
+            {children}
+        </span>
```

Comment:
Very interesting... 🤔 
Are there any consequences (good or bad) to these actually being `<span>`s, rather than bona fide, 100% organic, good ol' fashioned `<a>` elements?

Reply:
It mostly has to do with people that have special needs, but as well the docs say not using an `href` with an a tag isn't semantically correct.

Reply:
Got it.  Yeah, thinking about this more, basically the warnings are telling us that we're kind of misusing `<a>` elements.  So, our response is, okay then, we won't use `<a>` elements for that; we'll use `<span>` elements with `role="button"` to declare that they're interactive elements.  Makes sense.